src/core/manager.c

```diff
@@ -2749,6 +2749,10 @@ int manager_deserialize(Manager *m, FILE *f, FDSet *fds) {
                 r = unit_deserialize(u, f, fds);
                 if (r < 0)
                         goto finish;
+
+                r = unit_realize_cgroup(u);
```

Comment:
So, there has been a long-standing TODO list item, suggesting that we should enforce cgroup limits on daemon reload, hence I actually think @fbuihuu's original patch for this in this regard might make sense. But I figure this should be discussed in a different PR...

I think it would make a ton of sense if after a daemon-reload is issued we'd reapply as many bits of the configured settings we can. And cgroup limits are something which we can fix up easily, and we expose this even with "systemctl set-property"...